### ISSUE TITLE
[Feature Request] Change font size for normal text

### ISSUE THREAD

**Issue by tnlanh99** [microsoft/vscode-copilot-release#83]
I see there is a config setting to change font size, `chat.editor.fontSize`, but it only changes the code, not the others. Can you add a config for this?

<img width="1912" alt="image" src="https://github.com/microsoft/vscode-copilot-release/assets/38713308/1e52094f-327f-4135-884c-252cd3e1f01e">

**Comment by masakinihirota** [comment]
I agree with you, please give me the option to change the font size, please.

**Comment by hroos** [comment]
Yes please, that would be great!

**Comment by glekner** [comment]
please add an option for a custom font size!

**Comment by Mr-Zqr** [comment]
I agree. It would be a lot easier for me to read and chat with copilot if I can change the font bigger.

**Comment by roblourens** [comment]
If zooming doesn't work, then it sounds like you want the chat text to be larger than the rest of the UI text- can someone explain why they want that?

**Comment by Mr-Zqr** [comment]
> If zooming doesn't work, then it sounds like you want the chat text to be larger than the rest of the UI text- can someone explain why they want that?

I'm not sure if it's just because my screen is small or what. I found it difficult to read the text generated by copilot chat at the current font size. The text is so small and dense that I need to be close to the screen to read smoothly. However, the same font size works just fine for the rest of the UI text. I think it's because most of the UI only contains a few words and doesn't has long texts that need to be read and used as frequently as copilot chat.

**Comment by MangoMoe** [comment]
> If zooming doesn't work, then it sounds like you want the chat text to be larger than the rest of the UI text- can someone explain why they want that?

For me its because UI text is a couple of words or a sentence max (that only has to be read every once in a while), while chat often produces paragraphs of text, which takes longer to read, and leads to more strain.

**Comment by masakinihirota** [comment]
It is easiest to zoom in and out with the mouse wheel.

Solutions such as a button to copy GitHub Copilot chat output results to the main editor

Presbyopia Difficulty seeing small text

  "editor.mouseWheelZoom": true,
  "editor.fontSize": 20,

**Comment by liam-k** [comment]
> The UI has quite a lot of text in it in some places. I think the solution for this is to zoom, I don't want to have separate font settings for all parts of the UI
@roblourens 

Sorry, but that is just a bad call from an accessibility perspective. Zooming also increases the size of buttons, panels, icons, padding, sometimes in ways that are pretty useless. (I’m generally not a fan of how VSCode handles that.)

And if you have quite bad eyes, there’s a point where you can’t zoom the whole window enough to see enough but still be productive, because there’s just a bunch of useless padding. The way people with bad eyes (including me) use VSCode is by zooming the app window a bit, but zooming the editor font A LOT. Usually, you just have to read few things in the UI (like the occasional file name if you don’t navigate via CMD+P). It’s fine to just get closer to the screen for a second there. But if you’re trying to read a long text or explanation by Copilot, it’s absolutely horrible. 

I don’t know what you mean by "The UI has quite a lot of text in it in some places". Nowhere in the UI do you read what’s essentially an essay about a potentially complex topic for potentially several minutes. It’s entirely different.

Please reconsider this (or actually, implementing a UI font size across VSCode would be the best thing). It’s a really bad call ergonomically and in terms of accessibility. I can’t work like this – I’m back to copy pasting my code into ChatGPT. 🤷‍♂️ 

Attached is a screenshot of what I’m talking about.

![image](https://github.com/microsoft/vscode-copilot-release/assets/34057943/6239203c-1bde-416c-86b1-0e4ef1f269ef)

**Comment by healingdrawing** [comment]
who is the genius person, with super eyes of eagle probably , who does not think about control the font size for copilot chat?

Please show to us the github link of this epic sorcerer , to say thanx for his great kindness and responsibility to people around.

**Comment by datanetai** [comment]
Yes i need that feature too. Like other comment mention that zooming make everything else big too including menu etc so it is not feasible for me. My vision is weak so i already zoom and i want change the font size as well.

**Comment by roblourens** [comment]
We do have this issue still open on the vscode repo: https://github.com/microsoft/vscode/issues/519 And I think that would include this

**Comment by KrunoSaho** [comment]
Is it possible to include this feature before I go blind with age? Thanks.

**Comment by healingdrawing** [comment]
> Is it possible to include this feature before I go blind with age? Thanks.

actually, we do not know the real target of copilot, and ... it does not looks like the target is make user smarter and healthier , so ...

**Comment by afittro** [comment]
by the way, someone suggested earlier to toggle editor.mouseWheelZoom  to true. Doing so has this result: you scroll your mouse while hovering anywhere in vscode and it changes font smaller or larger. But with the Copilot Chat window open, **this only changes the code displayed** in the sidebar, and not any chat text from copilot or myself. It remains unusably small. All other font sizing options mentioned in this issue will only render the chat text large enough to read if the entire rest of the UI is horrifically large.

**Comment by irshadqemu** [comment]
Please add this feature. I have an eyesight issue, and the text in the GitHub co-pilot is too small to read for me.

**Comment by totkeks** [comment]
Fully agree, we need this. Because right now I'm feeling like this:

![Image](https://github.com/user-attachments/assets/c4ac76db-578f-4d53-9d35-3c78b5991427)

**Comment by totkeks** [comment]
> I'd suggest moving on to a better AI tool like Cursor, Cline, or Windsurf to anyone encountering this. You'll never look back. 

![9jpwjp.jpg](https://github.com/user-attachments/assets/274afb7a-ade0-4cb6-9a7e-6732248e029e)

**Comment by iwangbowen** [comment]
We need this setting
